demisto_sdk/commands/generate_test_playbook/test_playbook_generator.py

```diff
@@ -350,6 +367,10 @@ def run(self):
 
         test_playbook.add_task(create_end_task(test_playbook.task_counter))
 
+        if Path(self.test_playbook_yml_path).exists():
+            print_color(f'Warning: There already exists a test playbook at {self.test_playbook_yml_path}, '
+                        f'it will be overwritten.', LOG_COLORS.YELLOW)
+
         with open(self.test_playbook_yml_path, 'w') as yf:
             ryaml.dump(test_playbook.to_dict(), yf)
```

Comment:
Can you add here a print containing playbook was saved and its corresponding path? will be easier for user to follow where it was saved (and/or not miss it incase of users not aware of the defaults)

Reply:
There is one on line (old:356, new:377), is that what you meant?

Reply:
Missed it. Resolving